README.md

```diff
@@ -5,7 +5,7 @@
 
 The overall goal of the Vehicle Signal Specification (VSS) is to create a common understanding of vehicle signals in order to reach a “common language” independent of the protocol or serialisation format.
 
-Please find the official documentation at: [Vehicle Signal Specification](https://covesa.github.io/vehicle_signal_specification/).
+Generated documentation from latest commit on master branch is available at: [Vehicle Signal Specification Documentation](https://covesa.github.io/vehicle_signal_specification/).
```

Comment:
So this is the part worth discussing. I have now and then thought that term "official documentation" maybe is a bit misleading, as the source is in this repo and what you get on the site is just something generated from latest master-branch of this repo. So maybe this is a better phrasing, but I am open to other opinions/suggestions